hack/verify-kubebuilder.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
```

Comment:
Why is this called `verify-kubebuilder.sh`? Isn't this more about just verifying proto generation?

Reply:
I figured we could extend this to verify everything in kubebuilder, but I can make this proto specific

Reply:
:+1: that makes sense. I think I'd prefer a proto specific name for the file to better match the other verify-* script names. I may be misunderstanding this, but isn't proto generation unrelated to kubebuilder, and instead something we're adding on top with go-to-protobuf?

Reply:
That is a good point.. the kubebuild.mk also has format, vet, etc which aren't quite kubebuilder either which is why I placed this here. For now ill change the verify script

Reply:
Yeah, that's also a good point, confusing naming.